src/Microsoft.DotNet.Interactive.HttpRequest.Tests/Microsoft.DotNet.Interactive.HttpRequest.Tests.csproj

```diff
@@ -16,6 +16,18 @@
     <None Remove="TestResults\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="..\Microsoft.DotNet.Interactive.HttpRequestParser\*.cs" LinkBase="Parser" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="..\Microsoft.DotNet.Interactive.HttpRequestParser\obj\Debug\netstandard2.0\**" />
```

Comment:
Do we need this line? I think it may no longer be needed since we changed the `Compile` item on line 20 to only include *.cs files that are directly in the project folder (as opposed to recursively).